app/models/config.rb

```diff
@@ -65,4 +66,10 @@ def self.start_day
     start ||= "monday"
     start.downcase.to_sym
   end
+
+  def self.show_practical_support?
```

Comment:
ps I do really like the `Config.config_key` pattern and think it was a great introduction

Reply:
:D

Reply:
:+1: wooh!